.env.example

```
@@ -32,3 +32,7 @@ VUE_APP_GOOGLE_ANALYTICS_ID=
 
 # mixpanel
 VUE_APP_MIXPANEL_PROJECT_TOKEN= # get this from the Mixpanel project dashboard
+
+# chatwoot
+VUE_APP_CHATWOOT_WEBSITE= # URL of the deployed chatwoot app
```

we have used URL everywhere.
```suggestion
VUE_APP_CHATWOOT_URL= # URL of the deployed chatwoot app
```